website/docs/using_rf_in_ci_systems/ci/jenkins.md

```diff
@@ -124,16 +124,16 @@ This is related especially to the use of Browser library which requires installa
 ### Preparing the Jenkins agent
 
 These are steps that needs to be done by the person administrating the Jenkins agents.
-- Install all the tools that are needed for running the tests: nodejs, python 3.x, Robot Framework, Browser library
-- Set ```PLAYWRIGHT_BROWSERS_PATH``` to point where you want to install browsers
-- Execute ```rfbrowser init``` to get the base setup for Browser library. Note that you still need to execute it with Jenkins user as well, see below.
+1. Install all the tools that are needed for running the tests: nodejs, python 3.x, Robot Framework, Browser library
+1. Set ```PLAYWRIGHT_BROWSERS_PATH``` to point where you want to install browsers
+1. Execute ```rfbrowser init``` to get the base setup for Browser library. Note that you still need to execute it with Jenkins user as well, see below.
 
 ### Preparing the Jenkins configuration file
 
 These are steps you need to have in your Jenkins configuration. I recommend putting those as commands in the same shell command set together with test execution ("command to run your tests")
-- ```npm config set registry https://local-site-for-node-modules``` \<-- here you need a server that is providing all required node modules; if it doesn't exist proxy config might work as well
-- Additional commands that might be needed could be ```npm config set strict-ssl false``` or ```npm config set always-auth true```depending how the server is configured
-- Then run ```rfbrowser init --skip-browsers``` to avoid the installation of browsers that may appear hard over the firewall
-- ... and use instead the ones installed by root when the agent was initialised by pointing with ```PLAYWRIGHT_BROWSERS_PATH```` to the location defined
+1. ```npm config set registry https://local-site-for-node-modules``` \<-- here you need a server that is providing all required node modules; if it doesn't exist proxy config might work as well
+1. Additional commands that might be needed could be ```npm config set strict-ssl false``` or ```npm config set always-auth true```depending how the server is configured
+1. Then run ```rfbrowser init --skip-browsers``` to avoid the installation of browsers that may appear hard over the firewall
+1. ... and use instead the ones installed by root when the agent was initialised by pointing with ```PLAYWRIGHT_BROWSERS_PATH```` to the location defined
```

Comment:
There seems one extra backtick when closing ```PLAYWRIGHT_BROWSERS_PATH```` . This causes incorrect rendering:
![image](https://github.com/MarketSquare/robotframeworkguides/assets/24550370/9ccf2927-e2f3-488c-99a6-05dfd42c6cda)